lib/source-map.js

```diff
@@ -369,8 +386,10 @@ SourceMap.prototype._scanMappings = function(srcMap, sourcesOffset, namesOffset,
         inputMappings = inputMappings.slice(match[0].length);
       }
     }
-
   }
+
+  // this.decoder.resetColumn();
+  this.encoder.resetColumn();
```

Comment:
This seems like an important bugfix mixed in with otherwise semantics-preserving changes. What was the case where it was needed? A separate commit message, comment, or clearly linked test case would be good.